src/adapter.ts

```diff
@@ -0,0 +1,263 @@
+// Copyright 2021 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Helper, Model, FilteredAdapter} from 'casbin'
```

Comment:
Always add trailing ;

Use WebStorm to check all syntax

Reply:
I have used Prettier to check syntax in vscode

Reply:
follow the format of https://github.com/casbin/node-casbin